examples/nextjs-lit/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@lit-internal/examples-nextjs",
```

Comment:
We might want to grab `@lit-examples` so we can name the packages directly after the folder names, like `@lit-examples/nextjs-lit`

Reply:
done. renamed to `@lit-examples/nextjs` dropping `-lit` as per comment below.